### PR TITLE
adding sequence to the images animation

### PR DESCRIPTION
**Pull request by Sherifah**
# Description

Asynchronous JavaScript project using promises and asyn / await

This project is about adding Animation to three images by rotating the image and shrinking it until it disappears, one after the other. So when the first has finished we animate the second, and when the second has finished we animate the third.

Fixes # (issue)

# Type of change

- [ ] New feature (Animation)

## Change Details

- **What kind of change does this PR introduce?** (New feature)

- **What is the current behavior?** (Animating perfectly)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (gertie-sheshe) [inline .vscode/settings.json:3]: Were you trying to change live server port? Did this work? @Sherifah

**Review by gertie-sheshe — APPROVED** [review]
Great job!! 👏 🙌🏾🙌🏾🔥 @Sherifah I love that you have the Callback and Promises sections commented out. You can always come back to this PR to refer whenever you want to refresh yourself on the difference of all three.

File: .vscode/settings.json (hunk 1 above)
Comment (Sherifah) [reply .vscode/settings.json:3]: > Great job!! 👏 🙌🏾🙌🏾🔥 @Sherifah I love that you have the Callback and Promises sections commented out. You can always come back to this PR to refer whenever you want to refresh yourself on the difference of all three.

Smiles....I don't think I'll be able to master the callbacks so it's fortunate that it is no longer in vogue @gertie-sheshe

Comment (Sherifah) [reply .vscode/settings.json:3]: > Were you trying to change live server port? Did this work? @Sherifah

Hmmmm I don't know anything about that...I think that folder was included in the project file I downloaded from MDN @gertie-sheshe

Comment (Sherifah) [reply .vscode/settings.json:3]: @gertie-sheshe

**Pull request merged**